pyteal/ast/comment.py

```diff
@@ -0,0 +1,36 @@
+from typing import TYPE_CHECKING, Tuple
+
+from pyteal.ir import TealBlock, TealSimpleBlock
+from pyteal.ast.expr import Expr
+
+if TYPE_CHECKING:
+    from pyteal.compiler import CompileOptions
+
+
+class Comment(Expr):
+    def __init__(self, expr: Expr, comment: str):
+        self.expr = expr
+        self.comment = comment
+
+    def __teal__(self, options: "CompileOptions") -> Tuple[TealBlock, TealSimpleBlock]:
+        tb, tsb = self.expr.__teal__(options)
+
+        # TODO: make sure this doesnt cause issues
+        if len(tsb.ops) > 0:
+            tsb.ops[-1].args.append(f"// {self.comment}")
+        else:
+            tb.ops[-1].args.append(f"// {self.comment}")
```

Comment:
I'm not sure what the last paragraph means exactly. Do you mean add it as an op in the AVM or just make up an op in the ops.py?

Reply:
I meant to add a new op to PyTeal only. And yes, `ops.py` would be the place.

I suggest this because you'll be able to have `Comment` create lines with this approach, whereas currently you can only append to existing lines.

Reply:
Good point, I'll take a look at trying this

Reply:
https://github.com/algorand/pyteal/pull/410/files heres a draft that creates a new Op for comments, first attempt has it writing to the next line which I'm not sure is what we'd want?